model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -356,9 +360,29 @@ def copy_graph_with_ops(graph: Graph) -> Graph:
         if op_type in custom_ops:
             node = custom_ops[op_type](new_graph, op.attrs()).create_node()
         else:
-            assert op_type in Op.registered_ops, 'Operation {} not found in MO operations, ' \
-                                                 'please check it!'.format(op_type)
-            node = Op.get_op_class_by_name(op_type)(new_graph, op.attrs()).create_node()
+            if op_type not in Op.registered_ops:
+                log.warning('Operation {} not found in MO operations, please check it! '
+                            'Simple shape infer function is used'.format(op_type))
+                node = Op(new_graph, op.attrs()).create_node()
+                node['infer'] = Extender.const_shape_infer
+                if 'ir_data_attrs' in op:
+                    node['IE'] = [(
+                                    'layer',
+                                    [('id', lambda node: node.node), 'name', 'type', 'version'],
+                                    [
+                                        (
+                                            'data',
+                                            list(op.ir_data_attrs.keys()),
```

Comment:
will these keys be listed in expected order?

Reply:
We use usual dictionary here, so it is not guaranteed. But is this important to us? Do we provide layers attributes in IR in definite order?